coordinator/tributary/src/blockchain.rs

```diff
@@ -248,9 +262,9 @@ impl<D: Db, T: TransactionTrait> Blockchain<D, T> {
 
     for tx in &block.transactions {
       match tx.kind() {
-        TransactionKind::Provided(order) => {
-          self.provided.complete(&mut txn, order, tx.hash());
-        }
+        // We used to call the provided.complete() here
+        // but that is now moved to coordinator scanner.
```

Comment:
The `coordinator` should not be acknowledge by the `tributary`, a self-contained piece of code.